engine/build.gradle

```diff
@@ -96,7 +96,7 @@ dependencies {
     api group: 'org.codehaus.plexus', name: 'plexus-utils', version: '1.5.6'
 
     // Java magic
-    implementation group: 'net.java.dev.jna', name: 'jna-platform', version: '4.2.2'
+    implementation group: 'net.java.dev.jna', name: 'jna-platform', version: '4.5.2'
```

Comment:
There's a new major version of JNA 5.x out since October 2018, but the 4.5 series is sufficient to provide the [Resource](https://javadoc.io/doc/net.java.dev.jna/jna/4.5.2/com/sun/jna/platform/unix/Resource.html) API and I thought I could get away with doing less testing if I stuck to the same major version.

Manually keeping versions in sync between here and facade's build.gradle is a bother. Perhaps after clearing the jna-in-jopenvr obstacle we can revisit this and see if we can avoid that duplication.